packages/malloy/src/lang/ast/expressions/expr-func.ts

```diff
@@ -226,6 +226,12 @@ export class ExprFunc extends ExpressionDef {
         'sql_boolean',
       ].includes(func.name)
     ) {
+      if (!this.inExperiment('sql_functions')) {
```

Comment:
@christopherswenson - what do you think?

Reply:
Yep, I like adding this as an experiment. Though you should pass in `true` as a second parameter to `inExperiment` — otherwise it logs an error, as you can see in your test (it logs a generic error in addition to the custom one you're logging).